### PR TITLE
Faq & Tutorials part 1: Content and first design

### PR DESCRIPTION
**Pull request by TobiGr**
My time is quite limited right now, so I thought I'd be a good idea to get parts of my recent work online. In this way we can get feedback in an early state and I can change things which may cause problems. 

I intend to make four to five pull requests for the FAQ and tutorials thing: 

1. **Content and first design implementation (this PR):** 

    This PR refactored some important parts of the whole website to prepare the implementation of FAQs and tutorials: 

    - The navigation bar has be refactored completely and handles parameters to customize it. 

    - The tiles which were used for the homepage only are now also used for the FAQs. 

    The FAQ main page and some subpages are implemented. Because the most important thing (good content) is missing, the FAQ pages are labeled as BETA. Nevertheless this PR includes one tutorial about autoplay and seven FAQs: 

    - Download MP3 Files 

    - Download to SDCard fails 

    - Audio missing for certain downloads 

    - SoundCloud Beta problems @theScrabi I intend to label some issues as discussed in TeamNewPipe/NewPipe#1115 

    - Read YouTube comments 

    - YouTube / Google login 

    - What's New feed not sorted correctly 

     

2. **More tutorials and improved design** 

    Two more tutorials (Resize and close pop-up, open videos from other apps) and a better design for the FAQ landing page. I have written some more tutorials, but they need some more fine-tuning, so I guess this is going to come soon. 

3. **Even more tutorials** 

    Switch between players and Import/Export 

4. **Homepage integration** 

    Implementation as shown in https://github.com/TeamNewPipe/website/pull/72#issuecomment-388551794 with an improved concept and design 

    Final fixes and improvements to master the FAQs and grow out of the beta status. 

5. **App integration**

    App integration via welcome screen and tutorial hints on first or second use. See https://github.com/TeamNewPipe/NewPipe/issues/859

---

Some screenshots of the current implementation:

[FAQ & Tutorials main page](https://user-images.githubusercontent.com/17365767/43045755-ed435b2e-8dbe-11e8-82ae-dbf8a8bcae89.png)
[FAQ page: Import/Export & Download](https://user-images.githubusercontent.com/17365767/43045754-ed24ac88-8dbe-11e8-8252-786aa9588c8c.png)
[Tutorial page: autoplay](https://user-images.githubusercontent.com/17365767/43045753-ed08c0a4-8dbe-11e8-853d-fed29a30c94c.png)

---
Closes #71, Closes #72, First part of #69

### CODE REVIEW

**Comment by TheAssassin** [comment]
This looks _awesome_, @TobiGr. Thanks a lot!

Please feel free to merge this as soon as you're done, and I'll deploy it asap.

**Comment by TheAssassin** [comment]
Deploy in progress.

**Comment by TobiGr** [comment]
Thanks.
I just took a look at the design and I'd like to do something like this to reduce the grey background:
- [tutorial page](https://user-images.githubusercontent.com/17365767/43047427-67f133b0-8dd7-11e8-97b3-73e316b41d23.png)
 - [other page](https://user-images.githubusercontent.com/17365767/43047429-682862a4-8dd7-11e8-895c-a81ced8f91c2.png)
- [FAQ main page](https://user-images.githubusercontent.com/17365767/43047428-6810b4ba-8dd7-11e8-856a-fb5939c3d9ea.png): I am not sure whether the red tiles background works good with the category tiles. Maybe I need to make their background grey again. What do you think?

**Comment by TheAssassin** [comment]
I think that might be a bit "too much red".

By the way, the top menu on the homepage lacks the FAQ entry.

File: index.html [hunk 1]
```diff
@@ -62,6 +62,7 @@
                         <div id="navbar" class="navbar-collapse collapse navbar-right">
                             <ul class="nav navbar-nav">
                                 <li class="active"><a href="{{ site.baseurl }}/">Home</a></li>
+                                <li><a href="{{ site.baseurl }}/FAQ/">FAQ <span class="badge">BETA</span></a></li>
```
Comment (TobiGr) [inline index.html]: That is strange. I added it here.

Comment (TheAssassin) [reply index.html]: Sorry, my bad, was a caching issue.

Comment (TobiGr) [reply index.html]: I can see it when I open the website. I guess this is the browser cache...

**Comment by TheAssassin** [comment]
JFYI Sentry don't provide any services to us, they just maintain the tool. It's self-hosted.

**Pull request merged**